ClosedXML_Tests/Excel/Caching/SampleRepositoryTests.cs

```diff
@@ -1,10 +1,8 @@
+using ClosedXML.Excel;
 using ClosedXML.Excel.Caching;
 using NUnit.Framework;
-using System;
```

Comment:
Add back this `using`. It's needed for the Release configuration GC tests. Or explicitly use `System.WeakReference` in the test itself, so that we can omit the `using`.